tests/page-objects/AuthorDetailsPage.ts

```diff
@@ -0,0 +1,74 @@
+import { Selector, t } from 'testcafe';
+
+export class AuthorDetailsPage {
+    private readonly nextButton = Selector('.submission-wizard-next-button');
+
+    private readonly firstNameInput = Selector('.orcid-details__firstName');
+    private readonly lastNameInput = Selector('.orcid-details__lastName');
+    private readonly emailInput = Selector('.orcid-details__email');
+    private readonly institutionInput = Selector('.orcid-details__institution');
+
+    public async assertOnPage(): Promise<void> {
+        await t.expect(this.firstNameInput.visible).ok();
+        await t.expect(this.lastNameInput.visible).ok();
+        await t.expect(this.emailInput.visible).ok();
+        await t.expect(this.institutionInput.visible).ok();
+        await t.expect(this.nextButton.visible).ok();
+    }
+
+    public async populateForm(): Promise<void> {
+        await this.setFirstName('first');
+        await this.lastNameInput('last');
+        await this.setEmail('email@elifesciences.org');
+        await this.setInstitution('institution');
+    }
+
+    public async setEmail(input: string): Promise<void> {
```

Comment:
Can the setters all have a default, similar to the xpub page objects?

Reply:
Sure